docs/modules/language-reference/pages/index.adoc

```diff
@@ -5146,8 +5152,8 @@ It is defined by the presence of a `PklProject` file that amends the standard li
 Defining a project serves the following purposes:
 
 1. It allows defining common evaluator settings for Pkl modules within a logical project.
-2. It helps with managing <<packages,package>> dependencies for Pkl modules within a logical project.
-3. It enables packaging and sharing the contents of the project as a <<packages,package>>.
+2. It helps with managing <<package-asset-uri,package>> dependencies for Pkl modules within a logical project.
+3. It enables packaging and sharing the contents of the project as a <<package-asset-uri,package>>.
```

Comment:
These anchors are broken currently. I took an educated guess on where they should point, but I can revert or modify as needed.